src/main/resources/TaskManager/TaskManagerSheet.xml

```diff
@@ -68,11 +68,13 @@
               (((
                 (% class="form-group" %)
```

Comment:
Does it make sense in the application to have empty "form-group" divs? If not, then you probably need to move the if check higher to avoid unneeded empty markup.

Since I don`t know the exact logic of this UI I can not tell if this applies higher up the DOM. You could check if what I have mentioned above applies for other cases as well.